dev-lang/neko/neko-2.3.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="Neko is a high-level dynamically typed programming language"
+HOMEPAGE="https://nekovm.org/"
+
+if [[ "${PV}" == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/HaxeFoundation/${PN}.git"
+else
+	# 2.3.0 -> 2-3-0
+	MY_PV="${PV//./-}"
+	SRC_URI="https://github.com/HaxeFoundation/${PN}/archive/refs/tags/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/${PN}-${MY_PV}"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="apache mysql sqlite ssl"
+
+DEPEND="
+	apache? ( www-servers/apache:2 )
+	mysql?  ( dev-db/mysql:*       )
+	sqlite? ( dev-db/sqlite        )
+	ssl?    ( dev-libs/openssl     )
```

Comment:
Please don't do this, it is highly discouraged.
https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0101

Reply:
> Please don't do this, it is highly discouraged.
> https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0101

what do you mean? alignment of brackets?

the "QA: Ebuild file format" is only about code indentation. 
Does it matter also in case of not from the beginning of line to 1st character?

Reply:
Yes, 
"Do not attempt to align,"